### ISSUE TITLE
Error compiling the example

### ISSUE THREAD

**Issue by ramsestom** [danvick/flutter_form_builder#185]
Have this error when trying to compile the example:

> Compiler message:
> ../../SDK/.pub-cache/hosted/pub.dartlang.org/flutter_form_builder-3.5.3/lib/src/fields/form_builder_chips_input.dart:108:13: Error: No named parameter with the name 'onChipTapped'.
>             onChipTapped: widget.onChipTapped,
>             ^^^^^^^^^^^^
> ../../SDK/.pub-cache/hosted/pub.dartlang.org/flutter_chips_input-1.6.0/lib/src/chips_input.dart:12:3: Context: Found this candidate, but the arguments don't match.
>   ChipsInput({
>   ^

**Comment by ramsestom** [comment]
This bug seems fixed in flutter_form_builder >=3.6.1 but the problem is that flutter_localizations is incompatible with flutter_form_builder >=3.5.4:
> Because every version of flutter_localizations from sdk depends on intl 0.15.8 and flutter_form_builder >=3.5.4 depends on intl ^0.16.0, flutter_localizations from sdk is incompatible with flutter_form_builder >=3.5.4

So this is a real problem because it means that flutter_form_builder currently can't be used in any localized app using the standard flutter_localizations package. Do flutter_form_builder really need intl ^0.16.0, can't this be changed for a smaller intl minimal version?
In the mean time I would open an issue on flutter_localizations because they shouldn't enforce a specific version of their dependencies.

**Comment by lq-manh** [comment]
Hi @danvick,
Could you please fix this issue. I can't (I believe others like @ramsestom can't either) use this library in any way. I even tried with flutter_form_builder v3.3.x.

If intl v0.16.0 isn't that mandatory, could you downgrade it *for a while*? Or could you backport the fix with flutter_chips_input to flutter_form_builder v3.5.x? 

Thank you,

**Comment by danvick** [comment]
Hi @lq-manh,
I'll consider doing exactly that. In the meantime you could solve that with a dependency override in pubspec. Like this:
```
dependencies:
  ...
  flutter_form_builder: 3.6.1

dependency_overrides:
  intl: 0.15.8
```

**Comment by danvick** [comment]
> Have this error when trying to compile the example:
> 
> > Compiler message:
> > ../../SDK/.pub-cache/hosted/pub.dartlang.org/flutter_form_builder-3.5.3/lib/src/fields/form_builder_chips_input.dart:108:13: Error: No named parameter with the name 'onChipTapped'.
> > onChipTapped: widget.onChipTapped,
> > ^^^^^^^^^^^^
> > ../../SDK/.pub-cache/hosted/pub.dartlang.org/flutter_chips_input-1.6.0/lib/src/chips_input.dart:12:3: Context: Found this candidate, but the arguments don't match.
> > ChipsInput({
> > ^

Kindly use the latest version of the package

**Comment by danvick** [comment]
Fixed in v3.7.0. Feel free to reopen in case of any more feedback.

**Comment by malikdoksoz** [comment]
Can you update intl 16.0 

> Because flutter_form_builder 3.7.2 depends on intl ^0.15.8 and no versions of flutter_form_builder match >3.7.2 <4.0.0, flutter_form_builder ^3.7.2 requires intl ^0.15.8.
